Compilation fails

```
cargo generate --git https://github.com/mciantyre/teensy4-rs-template --name hello-world
cd .\hello-world\
cargo build --release
[...]
error[E0433]: failed to resolve: could not find `interrupt` in `rt`
  --> C:\Users\X\.cargo\git\checkouts\teensy4-rs-32cd33027d5b365b\ca100f3\src\usb.rs:89:14
   |
89 | #[crate::rt::interrupt]
   |              ^^^^^^^^^ could not find `interrupt` in `rt`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `teensy4-bsp`.

To learn more, run the command again with --verbose.
```

Presumably caused by https://github.com/mciantyre/teensy4-rs/commit/c062ff26eca247f7f78c5b69be455d7656a04aca

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,6 +12,8 @@ embedded-hal = "0.2.3"
 [dependencies.teensy4-bsp]
 git = "https://github.com/mciantyre/teensy4-rs.git"
 
-[[bin]]
-name = "{{crate_name}}"
-path = "src/main.rs"
+# Patch `cortex-m-rt` for reasons described here:
+# https://github.com/mciantyre/teensy4-rs#runtime
+[patch.crates-io.cortex-m-rt]
+git = "https://github.com/mciantyre/teensy4-rs"
+branch = "master"

